src/python/director/viewerclient.py

```diff
@@ -205,13 +206,20 @@ def __init__(self, lcm=None):
         if lcm is None:
             lcm = LCM()
         self.lcm = lcm
+        self.client_id_file = tempfile.NamedTemporaryFile()
```

Comment:
i don't love this.  what about `os.getpid()` plus a global counter variable, or id(self).  (trying to think of short strings, uuid is probably the best long string)